config/spring.rb

```diff
@@ -0,0 +1,6 @@
+%w[
+  .ruby-version
+  .rbenv-vars
+  tmp/restart.txt
+  tmp/caching-dev.txt
+].each { |path| Spring.watch(path) }
```

Comment:
Should we kill this in favor of `bootsnap` since we're upgrading to rail 5.2

Reply:
Apparently Rails upstream is injecting both 🤷‍♂️

Reply:
_sigh_